portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,8 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+img{
+    width: 25%;
+    height: auto;
+}
```

Comment:
This was also added in your other branch.  Instead of duplicating changes across branches, you can:
1) Branch this off of your other branch (instead of master)
2) Finish the pull request for the other branch and then rebase this one
3) Abandon one of these and make these changes in multiple commits for the same branch
4) Remove the duplicate changes.  They'll still end up in the master branch, but if they aren't necessary for this change you should remove them

Reply:
In order to make a new branch from this branch, do i just use "git checkout -b [branch name]"?

Reply:
In order to branch this off of my Pictures branch, do I use "git checkout -b [branch name]"?